docs/sources/alerting/unified-alerting/alerting-rules/alert-annotation-label.md

```diff
@@ -33,28 +33,28 @@ The following template variables are available when expanding annotations and la
+| ------------------ | -------------------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| humanize           | number or string           | string                                              | Converts a number to a more readable format, using metric prefixes.                                                                         |
+| humanize1024       | number or string           | string                                              | Like humanize, but uses 1024 as the base rather than 1000.                                                                                  |
+| humanizeDuration   | number or string           | string                                              | Converts a duration in seconds to a more readable format.                                                                                   |
+| humanizePercentage | number or string           | string                                              | Converts a ratio value to a fraction of 100.                                                                                                |
+| humanizeTimestamp  | number or string           | string                                              | Converts a Unix timestamp in seconds to a more readable format.                                                                             |
+| title              | string                     | string                                              | strings.Title, capitalises first character of each word.                                                                                    |
+| toUpper            | string                     | string                                              | strings.ToUpper, converts all characters to upper case.                                                                                     |
+| toLower            | string                     | string                                              | strings.ToLower, converts all characters to lower case.                                                                                     |
+| match              | pattern, text              | boolean                                             | regexp.MatchString Tests for a unanchored regexp match.                                                                                     |
+| reReplaceAll       | pattern, replacement, text | string                                              | Regexp.ReplaceAllString Regexp substitution, unanchored.                                                                                    |
+| graphLink          | expr                       | JSON string with `"expr"` and `"datasource"` fields | Returns path to graph view in the [Explore](https://grafana.com/docs/grafana/latest/explore/) page for the given expression and datasource. |
```

Comment:
"JSON String ..." I wonder if that is what it actually returns. According to the code, it returns a relative URI to Explore page

Reply:
I got those mixed up, that should go in the "argument" column 🤦